arcgis-ios-sdk-samples/Features/Feature layer selection/FLSelection.storyboard

```diff
@@ -1,35 +1,54 @@
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
         <!--Selection View Controller-->
         <scene sceneID="9Cx-Fs-l9y">
             <objects>
-                <viewController id="PIk-JH-uY3" customClass="FLSelectionViewController" customModule="arcgis_ios_sdk_samples" customModuleProvider="target" sceneMemberID="viewController">
+                <viewController id="PIk-JH-uY3" customClass="FLSelectionViewController" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target" sceneMemberID="viewController">
                     <view key="view" contentMode="scaleToFill" id="pct-Is-DQi">
-                        <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
+                        <rect key="frame" x="0.0" y="0.0" width="414" height="896"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Jek-wa-W5c" customClass="AGSMapView">
-                                <rect key="frame" x="0.0" y="20" width="375" height="647"/>
-                                <color key="backgroundColor" red="1" green="1" blue="1" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
+                                <rect key="frame" x="0.0" y="44" width="414" height="852"/>
+                            </view>
+                            <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="AIN-Oz-V0e">
+                                <rect key="frame" x="0.0" y="44" width="414" height="21"/>
+                                <subviews>
+                                    <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Selected feature count shown here." textAlignment="center" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" translatesAutoresizingMaskIntoConstraints="NO" id="MSB-XK-96k">
+                                        <rect key="frame" x="0.0" y="0.0" width="414" height="21"/>
+                                        <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                        <nil key="textColor"/>
+                                        <nil key="highlightedColor"/>
+                                    </label>
+                                </subviews>
+                                <color key="backgroundColor" white="1" alpha="0.59999999999999998" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                <constraints>
+                                    <constraint firstItem="MSB-XK-96k" firstAttribute="top" secondItem="gTA-cM-d3f" secondAttribute="top" id="0NH-nf-VjW"/>
```

Comment:
Something seems odd about these constraints. The label's top and bottom should be constrained to the top and bottom of its superview, not the safe area. Right now there are no constraints to the bottom of the superview, meaning that its height is not well defined.

Reply:
This is similar to one of previous discussions: https://github.com/Esri/arcgis-runtime-samples-ios/pull/837#discussion_r403031441

I want to create a "HUD" atop, without letting the label stretch into the safe area, while the translucent background should snap to superview's leading and trailing anchors. Even though currently status messages never overflow the width, I still want to play it safely.

![image](https://user-images.githubusercontent.com/9660181/89322501-23c7d480-d639-11ea-8199-dd818e815436.png)

So what I did is to embed the label in a view, snap the label to the safe area of the view, and set the view's leading and trailing to superview.

And for the top constraint of the view, it is to snap to top of map view. The height of this wrapping view is decided by the height of the label, so there is no bottom anchor for it. Do you think it's preferable to give an explicit height to it?

Reply:
> Do you think it's preferable to give an explicit height to it?

Yes and no. Not by means of an explicit height constraint, but there should be an unbroken line of constraints from one side of each view to the other that unambiguously determine both the size and position of the view. For the wrapping view, as far as I can tell, there are no constraint(s) that determines its height. To fix that, I would add a constraint from the top of the label to the top of the wrapper view and another constraint from the bottom of the label to the bottom of the wrapper view, replacing the current top and bottom constraints for the label. That way the wrapper view's height will be equal to the height of the label, which is determined by the label's intrinsic content size (which is based on the label's text).

Reply:
Addressed. Hope this time it looks right... 😄